helpers/updater.js

```diff
@@ -14,6 +17,18 @@ const divider = '---';
 
 console.log('Updater is initiated.');
 
+// Make sure the options used here are the same as used in `hexo`.
+marked.setOptions({
```

Comment:
Can we have this options in a config file and use it in both places?

Reply:
@molant Yes, just made the change.